content/hackathon/prd/tactile-sidewalk-wayfinding-strip.md

```diff
@@ -0,0 +1,54 @@
+---
+title: "Hackathon Project #7: Tactile Sidewalk Wayfinding Strip"
+---
+
+## Idea
+
+A continuous tactile path on all sidewalks that provides wayfinding to and contextual, place-based information about the nearest transit stop.
+
+## Use cases
+
+1. Someone new to the neighborhood who is blind or partially sighted follows a path from their hotel to a nearby bus stop.
+1. Someone who’s looking down at their phone notices that a crosswalk is coming up because of the tactile cue they’re stepping on.
+
+## This prototype must include
+
+1. Indicate the direction the nearest transit stop and provide warnings about crosswalks beginning / ending.
+1. Must be able to integrate with the existing modular hexagonal paver system. These should be per-paver systems (either toppers for pavers or actually cast into the pavers themselves - this last option would required a limited number of types of wayfinding pavers to be defined).

```

Comment:
Typo 'required' instead of 'require: .... this last options would require a limited ....